src/metabase/util/password.clj

```diff
@@ -88,6 +88,12 @@
        (or (= (config/config-kw :mb-password-complexity) :weak)
            (is-uncommon? password))))
 
+(def ^:private default-bcrypt-work-factor
+  "Default work factor used for hashing passwords with BCrypt. Intentionally minimal for tests to reduce testing time."
+  (if config/is-test?
+    4
+    10))
```

Comment:
I think it's worth commenting in the code that 4 is the minimum supported value.